hawc/apps/epiv2/urls.py

```diff
@@ -7,6 +7,12 @@
 router.register(r"assessment", api.EpiAssessmentViewset, basename="assessment")
 router.register(r"design", api.Design, basename="design")
 router.register(r"metadata", api.Metadata, basename="metadata")
+router.register(r"chemical", api.Chemical, basename="chemical")
+router.register(r"exposure", api.Exposure, basename="exposure")
+router.register(r"exposurelevel", api.ExposureLevel, basename="exposurelevel")
```

Comment:
use hyphens `exposure-level` for this and other multiwords?  i don't think they need to be raw strings

Reply:
I went back and forth on this when setting them up - if I recall, my rationale was that in the urlpatterns defined later in that file, they weren't hyphen delimited, and I wanted to keep the API ones consistent with that? Anyway - for the new API routes, switched "exposurelevel" -> "exposure-level", etc.

```
    # exposure level
    path(
        "exposurelevel/<int:pk>/create/",
        views.ExposureLevelViewset.as_view(),
        {"action": "create"},
        name="exposurelevel-create",
    ),
# .....
    # adjustment factor
    path(
        "adjustmentfactor/<int:pk>/create/",
        views.AdjustmentFactorViewset.as_view(),
        {"action": "create"},
        name="adjustmentfactor-create",
    ),
 # etc.
```